pkg/view/templates/layout/base.html

```diff
@@ -61,33 +61,6 @@
                     <span class="ms-3">API Keys</span>
```

Comment:
Any reason you're leaving this link here? (The page doesn't do anything.)

Reply:
nope, gone